pkgs/by-name/co/comet/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication {
+  pname = "comet";
+  version = "unstable-2023-12-09";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "imLinguin";
+    repo = "comet";
+    rev = "17cc4a739de0f25e06fbaaca06047207b5048b94";
+    hash = "sha256-QXNHTSsz93VM7u4yWtX5oKQb4ZpFgIqnj7rXyXohDEM=";
+  };
+
+  nativeBuildInputs = [ python3.pkgs.pdm-backend ];
```

```suggestion
  nativeBuildInputs = with python3.pkgs; [ pdm-backend ];
```
This will make diffs easier to read when adding more deps.